sdk/identity/identity/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.1.0-preview.4 (Unreleased)
 
+- Added support for the `IDENTITY_ENDPOINT` and `IDENTITY_SECRET` environment variables.
```

Comment:
Might help to be a little more explicit here:

```suggestion
- Added support for the `IDENTITY_ENDPOINT` and `IDENTITY_SECRET` environment variables when using `ManagedIdentityCredential` on Azure App Service.
```

Reply:
Thank you!